pkg/pillar/cas/containerd.go

```diff
@@ -47,6 +47,51 @@ type containerdCAS struct {
 	ctrdClient *containerd.Client
 }
 
+// SnapshotUsage returns current usage of snapshot in bytes
+// if parents defined also adds all parents usage
+func (c *containerdCAS) SnapshotUsage(snapshotID string, parents bool) (int64, error) {
```

Comment:
What exactly are the parents of the snapshot? I see you always set it but I don't understand what it is and whether one would ever not want to include the parents.